--- kdeplasma-addons-4.3.69svn1027298/applets/pastebin/pastebin.cpp.orig	2009-09-23 21:22:34.000000000 +0200
+++ kdeplasma-addons-4.3.69svn1027298/applets/pastebin/pastebin.cpp	2009-10-08 01:10:11.000000000 +0200
@@ -460,7 +460,7 @@
             break;
 
         case Pastebin::PASTEBINCOM:
-           serverAddress = cg.readEntry("pastebincom", "http://pastebin.com");
+           serverAddress = cg.readEntry("pld.pastebincom", "http://pld.pastebin.com");
            break;
     }
 
@@ -810,7 +810,7 @@
                 break;
 
             case Pastebin::PASTEBINCOM:
-                ops.writeEntry("server", cg.readEntry("pastebincom", "http://pastebin.com"));
+                ops.writeEntry("server", cg.readEntry("pld.pastebincom", "http://pld.pastebin.com"));
                 break;
         }
 
