diff -ur kdeplasma-addons-4.3.90/applets/pastebin/pastebin.cpp kdeplasma-addons-4.3.90.new/applets/pastebin/pastebin.cpp
--- kdeplasma-addons-4.3.90/applets/pastebin/pastebin.cpp	2009-12-17 17:38:19.000000000 +0100
+++ kdeplasma-addons-4.3.90.new/applets/pastebin/pastebin.cpp	2010-01-13 13:09:05.205990344 +0100
@@ -466,7 +466,7 @@
             break;
 
         case Pastebin::PASTEBINCOM:
-           serverAddress = cg.readEntry("pastebincom", "http://pastebin.com");
+           serverAddress = cg.readEntry("pastebincom", "http://pld.pastebin.com");
            break;
     }
 
@@ -818,7 +818,7 @@
                 break;
 
             case Pastebin::PASTEBINCOM:
-                ops.writeEntry("server", cg.readEntry("pastebincom", "http://pastebin.com"));
+                ops.writeEntry("server", cg.readEntry("pastebincom", "http://pld.pastebin.com"));
                 break;
         }
 
diff -ur kdeplasma-addons-4.3.90/applets/pastebin/pastebin.h kdeplasma-addons-4.3.90.new/applets/pastebin/pastebin.h
--- kdeplasma-addons-4.3.90/applets/pastebin/pastebin.h	2009-12-05 00:09:42.000000000 +0100
+++ kdeplasma-addons-4.3.90.new/applets/pastebin/pastebin.h	2010-01-13 13:28:15.798996766 +0100
@@ -70,7 +70,7 @@
         Sending = 8      /* Sending data to the server, waiting for reply */
     };
 
-    enum textServers { PASTEBINCA, PASTEBINCOM };
+    enum textServers { PASTEBINCOM, PASTEBINCA };
     enum imageServers { IMAGEBINCA, IMAGESHACK, SIMPLESTIMAGEHOSTING };
 
 public slots:
diff -ur kdeplasma-addons-4.3.90/dataengines/pastebin/backends/pastebincom.h kdeplasma-addons-4.3.90.new/dataengines/pastebin/backends/pastebincom.h
--- kdeplasma-addons-4.3.90/dataengines/pastebin/backends/pastebincom.h	2009-09-04 13:31:16.000000000 +0200
+++ kdeplasma-addons-4.3.90.new/dataengines/pastebin/backends/pastebincom.h	2010-01-13 13:10:03.552445605 +0100
@@ -27,7 +27,7 @@
 #include <kio/global.h>
 #include <kio/job.h>
 
-#define PASTEBINCOM_SERVER "http://pastebin.com"
+#define PASTEBINCOM_SERVER "http://pld.pastebin.com"
 
 class PastebinCOMServer : public PastebinServer
 {
diff -ur kdeplasma-addons-4.3.90/dataengines/pastebin/pastebinservice.h kdeplasma-addons-4.3.90.new/dataengines/pastebin/pastebinservice.h
--- kdeplasma-addons-4.3.90/dataengines/pastebin/pastebinservice.h	2009-10-15 18:34:11.000000000 +0200
+++ kdeplasma-addons-4.3.90.new/dataengines/pastebin/pastebinservice.h	2010-01-13 13:28:35.522328387 +0100
@@ -38,7 +38,7 @@
     Q_OBJECT
 
 public:
-    enum textServers { PASTEBINCA, PASTEBINCOM };
+    enum textServers { PASTEBINCOM, PASTEBINCA };
     enum imageServers { IMAGEBINCA, IMAGESHACK, SIMPLESTIMAGEHOSTING };
 
     PastebinService(PastebinEngine *engine);
