Index: dataengines/comic/comicproviderwrapper.cpp
===================================================================
--- dataengines/comic/comicproviderwrapper.cpp	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ dataengines/comic/comicproviderwrapper.cpp	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -573,6 +573,28 @@
         case StringIdentifier:
             break;
         }
+    //check if the nextIdentifier is correct
+    } else if ( !mNextIdentifier.isNull() ) {
+        //no nextIdentifier if mIdentifier == mLastIdentifier or if no identifier has been specified
+        switch ( identifierType() ) {
+        case DateIdentifier:
+            if ( ( !mLastIdentifier.isNull() && ( mIdentifier.toDate() == mLastIdentifier.toDate() ) ) ||
+                 !mIdentifierSpecified ) {
+                return false;
+            }
+            break;
+        case NumberIdentifier:
+            if ( ( !mLastIdentifier.isNull() && ( mIdentifier.toInt() == mLastIdentifier.toInt() ) ) ||
+                 !mIdentifierSpecified ) {
+                return false;
+            }
+            break;
+        case StringIdentifier:
+            if ( !mIdentifierSpecified ) {
+                return false;
+            }
+            break;
+        }
     }
     return mNextIdentifier;
 }
@@ -598,6 +620,22 @@
         case StringIdentifier:
             break;
         }
+    } else if ( !mPreviousIdentifier.isNull() ) {
+        //no previousIdentifier if mIdentifier == mFirstIdentifier
+        switch ( identifierType() ) {
+        case DateIdentifier:
+            if ( !mFirstIdentifier.isNull() && ( mIdentifier.toDate() == mFirstIdentifier.toDate() ) ) {
+                return false;
+            }
+            break;
+        case NumberIdentifier:
+            if ( !mFirstIdentifier.isNull() && ( mIdentifier.toInt() == mFirstIdentifier.toInt() ) ) {
+                return false;
+            }
+            break;
+        case StringIdentifier:
+            break;
+        }
     }
     return mPreviousIdentifier;
 }
Index: dataengines/comic/plasma_comicprovider.desktop
===================================================================
--- dataengines/comic/plasma_comicprovider.desktop	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ dataengines/comic/plasma_comicprovider.desktop	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -17,7 +17,7 @@
 Comment[gl]=Extensión para o motor de Cómics de Plasma
 Comment[he]=פלאגין של מנוע הקומיקסים עבור Plasma
 Comment[hu]=Modul a Plasma képregényrajzolóhoz
-Comment[it]=Plugin per il Motore fumetti per Plasma
+Comment[it]=Estensione per il motore di fumetti di Plasma
 Comment[ja]=Plasma コミックエンジンのためのプラグイン
 Comment[km]=កម្មវិធី​ជំនួយ​សម្រាប់​ម៉ាស៊ីន​បំប្លែង Plasma
 Comment[ko]=Plasma 만화 엔진 플러그인
Index: dataengines/comic/plasma-dataengine-comic.desktop
===================================================================
--- dataengines/comic/plasma-dataengine-comic.desktop	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ dataengines/comic/plasma-dataengine-comic.desktop	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -13,7 +13,7 @@
 Name[ga]=Stiallchartúin
 Name[gl]=Banda deseñada
 Name[he]=רצועות הקומיקס
-Name[it]=Striscie a fumetti
+Name[it]=Strisce di fumetti
 Name[ja]=コミック・ストリップ
 Name[km]=រឿង​កំប្លែង
 Name[ku]=Pirtûkên Qerfî
Index: dataengines/comic/comic.cpp
===================================================================
--- dataengines/comic/comic.cpp	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ dataengines/comic/comic.cpp	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -173,7 +173,7 @@
     // different comic -- with no error yet -- has been chosen, old error is invalidated
     QString temp = mIdentifierError.left( mIdentifierError.indexOf( ':' ) + 1 );
     if ( !mIdentifierError.isEmpty() && provider->identifier().indexOf( temp ) == -1 ) {
-        mIdentifierError = QString();
+        mIdentifierError.clear();
     }
 
     // store in cache if it's not the response of a CachedProvider,
Index: applets/showdashboard/plasma-applet-showdashboard.desktop
===================================================================
--- applets/showdashboard/plasma-applet-showdashboard.desktop	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ applets/showdashboard/plasma-applet-showdashboard.desktop	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -13,7 +13,7 @@
 Name[ga]=Taispeáin Painéal na nGiuirléidí
 Name[gl]=Mostrar o widget do cadro de control
 Name[he]=הצג את המכתבה של Plasma
-Name[it]=Mostra la Dashboard dei widget
+Name[it]=Mostra la dashboard degli oggetti
 Name[ja]=ウィジェットのダッシュボードを表示
 Name[km]=បង្ហាញផ្ទាំង​គ្រប់គ្រង​ធាតុក្រាហ្វិក
 Name[ku]=Kontrolkerê Alavan Nîşan Bide
@@ -66,8 +66,8 @@
 Comment[ro]=Arată tabloul de bord cu controale Plasma deasupra celorlalte ferestre
 Comment[ru]=Показывает прибоную доску Plasma поверх всех окон
 Comment[sl]=Prikaži armatruno ploščo gradnikov Plasme pred drugimi okni
-Comment[sr]=Инструмент-табла Плазма виџета изнад других прозора
-Comment[sr@latin]=Instrument-tabla Plasma vidžeta iznad drugih prozora
+Comment[sr]=Инструмент-табла плазма виџета изнад других прозора
+Comment[sr@latin]=Instrument-tabla plasma vidžeta iznad drugih prozora
 Comment[sv]=Visa den grafiska instrumentpanelkomponenten i Plasma ovanför andra fönster
 Comment[th]=แสดงแดชบอร์ดบรรจุวิดเจ็ตของพลาสมา ให้อยู่เหนือหน้าต่างอื่น ๆ
 Comment[tr]=Plasma Programcık kontrol panelini diğer pencerelerin üzerinde göster
Index: applets/comic/plasma-comic-default.desktop
===================================================================
--- applets/comic/plasma-comic-default.desktop	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ applets/comic/plasma-comic-default.desktop	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -16,7 +16,7 @@
 Name[gl]=Banda deseñada
 Name[he]=קומיקס
 Name[hu]=Képregény
-Name[it]=Striscia a fumetti
+Name[it]=Striscia di fumetti
 Name[ja]=コミック・ストリップ
 Name[km]=យក​កំប្លែង​ចេញ
 Name[ko]=만화 조각
@@ -56,7 +56,7 @@
 Comment[ga]=Taispeáin stiallchartún ón Idirlíon
 Comment[gl]=Mostra bandas deseñadas de internet
 Comment[he]=מציג קומיקסים נבחרים מהאינטרנט
-Comment[it]=Vedi una striscia a fumetti da internet.
+Comment[it]=Vedi una striscia di fumetti da Internet.
 Comment[ja]=インターネット上のコミック・ストリップを表示します
 Comment[km]=មើល​រឿង​កំប្លែង​ពី​អ៊ីនធឺណិត
 Comment[ku]=Ji Torê pirtûkên qerfî bibîne
Index: applets/comic/comic.cpp
===================================================================
--- applets/comic/comic.cpp	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ applets/comic/comic.cpp	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -660,6 +660,7 @@
 {
     mEngine = dataEngine( "comic" );
 
+    setConfigurationRequired( mComicIdentifier.isEmpty() );
     if ( !mComicIdentifier.isEmpty() && mEngine && mEngine->isValid() ) {
         const QString identifier = mComicIdentifier + ':' + identifierSuffix;
         mEngine->disconnectSource( identifier, this );
@@ -696,7 +697,7 @@
 {
     mActionGoFirst->setVisible( !mFirstIdentifierSuffix.isEmpty() );
     mActionGoFirst->setEnabled( !mPreviousIdentifierSuffix.isEmpty() );
-    mActionGoLast->setEnabled( !mNextIdentifierSuffix.isEmpty() );
+    mActionGoLast->setEnabled( true );//always enabled to have some kind of refresh action
 }
 
 void ComicApplet::slotSaveComicAs()
Index: applets/lancelot/app/src/Messages.sh
===================================================================
--- applets/lancelot/app/src/Messages.sh	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ applets/lancelot/app/src/Messages.sh	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -1,4 +1,4 @@
 #! /usr/bin/env bash
 $EXTRACTRC `find . -name \*.ui` >> rc.cpp || exit 11
-$XGETTEXT *.cpp -o $podir/lancelot.pot
+$XGETTEXT `find . -name \*.cpp` -o $podir/lancelot.pot
 rm -f rc.cpp
Index: applets/twitter/plasma-twitter-default.desktop
===================================================================
--- applets/twitter/plasma-twitter-default.desktop	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ applets/twitter/plasma-twitter-default.desktop	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -46,7 +46,7 @@
 Comment[ga]=Nuashonrú do stádas Twitter agus taispeáin do líne ama.
 Comment[gl]=Anova o seu estado en Twitter e mostra a súa liña de tempo.
 Comment[he]=עדכן את סטטוס ה Twitter שלך והצג את ציר הזמן שלך
-Comment[it]=Aggiornare il tuo stato di Twitter e mostra la tua cronologia.
+Comment[it]=Aggiorna lo stato di Twitter e mostra la cronologia.
 Comment[ja]=Twitter の状態を更新し、あなたのタイムラインを表示します
 Comment[km]=ធ្វើ​ឲ្យ​ស្ថានភាព Twitter របស់​អ្នក​ទាន់សម័យ និង​បង្ហាញ​ពេលវេលា​របស់​អ្នក ។
 Comment[ku]=Rewşên Twitter rojane bike û rêzika-demê nîşan bide
Index: applets/kolourpicker/kolourpicker.cpp
===================================================================
--- applets/kolourpicker/kolourpicker.cpp	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ applets/kolourpicker/kolourpicker.cpp	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -124,6 +124,7 @@
     QPixmap pix(size);
     pix.fill(QColor(0,0,0,0));
     QPainter p(&pix);
+    p.setRenderHint(QPainter::Antialiasing, true);
     paint(&p, pix.rect(), mode, state);
     p.end();
     return pix;
Index: applets/konqprofiles/plasma-applet-konqprofiles.desktop
===================================================================
--- applets/konqprofiles/plasma-applet-konqprofiles.desktop	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ applets/konqprofiles/plasma-applet-konqprofiles.desktop	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -55,7 +55,7 @@
 Comment[gl]=Enumera e inicia perfís de Konqueror
 Comment[he]=הצג והפעל פרופילים של Konqueror
 Comment[hu]=Indítható Konqueror-profilok
-Comment[it]=Lista e lanciatore dei profili di Konqueror
+Comment[it]=Elenca ed esegue i profili di Konqueror
 Comment[ja]=Konqueror のプロファイルを起動します
 Comment[km]=រាយ និង​ចាប់ផ្ដើម​ទម្រង់​របស់ Konqueror
 Comment[ku]=Profîlên Konqueror lîste bike û bide destpêkirin
Index: applets/life/plasma-applet-life.desktop
===================================================================
--- applets/life/plasma-applet-life.desktop	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ applets/life/plasma-applet-life.desktop	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -27,7 +27,7 @@
 Name[sr]=живот
 Name[sr@latin]=život
 Name[th]=ไลฟ์
-Name[uk]=Термін використання
+Name[uk]=Життя
 Name[wa]=Vicåreye
 Name[x-test]=xxLifexx
 Name[zh_CN]=生命
@@ -45,7 +45,7 @@
 Comment[ga]= Feidhmchláirín Conway's Game of Life
 Comment[gl]=Applet do xogo da vida de Conway
 Comment[he]=משחק החיים של Conway
-Comment[it]=Appet del Gioco della vita di Conway
+Comment[it]=Applet del gioco della vita di Conway
 Comment[ja]=ライフゲーム (Conway's Game of Life) アプレット
 Comment[km]= អាប់ភ្លេត​ជីវិត​ល្បែង​របស់ Conway
 Comment[ku]=Sepanoka Lîstika Jiyanê ya Conway 
Index: applets/timer/plasma-applet-timer.desktop
===================================================================
--- applets/timer/plasma-applet-timer.desktop	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ applets/timer/plasma-applet-timer.desktop	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -14,6 +14,7 @@
 Name[ga]=Uaineadóir
 Name[gl]=Temporizador
 Name[he]=קוצב זמן
+Name[it]=Temporizzatore
 Name[ja]=タイマー
 Name[km]=កម្មវិធី​កំណត់​ពេលវេលា
 Name[lt]=Laikmatis
Index: applets/leavenote/plasma-applet-leavenote.desktop
===================================================================
--- applets/leavenote/plasma-applet-leavenote.desktop	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ applets/leavenote/plasma-applet-leavenote.desktop	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -56,7 +56,7 @@
 Comment[gl]=Deixa notas para os usuarios mentras estásn ausentes
 Comment[he]=משאיר פתקים בשביל משתמשים בזמן שהם לא נמצאים
 Comment[hu]=Távolléti üzenet
-Comment[it]=Lascia una nota per gli utenti mentre sono via
+Comment[it]=Lascia una nota per gli utenti assenti
 Comment[ja]=離席中のユーザにメモを残します
 Comment[km]=ទុក​ចំណាំ​សម្រាប់​អ្នកប្រើ​ខណៈពេល​ដែល​ពួកគេ​ចាក​ឆ្ងាយ
 Comment[ku]=Ji bo bikarhênerên niha ne li vir in nîşeyan binivîse
Index: applets/calculator/calculator.cpp
===================================================================
--- applets/calculator/calculator.cpp	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ applets/calculator/calculator.cpp	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -336,7 +336,7 @@
 
     if (!inputText.contains(KGlobal::locale()->decimalSymbol())) {
       //If there is no decimal, then we need to reformat the number
-      double currentValue = inputText.toDouble();
+      double currentValue = KGlobal::locale()->readNumber(inputText);
       QString localizedString = KGlobal::locale()->formatNumber(currentValue, 0);
       mOutputDisplay->setText(localizedString);
     } else {
@@ -366,7 +366,7 @@
 
 void CalculatorApplet::slotAddClicked()
 {
-    double currentValue = inputText.toDouble();
+    double currentValue = KGlobal::locale()->readNumber(inputText);
 
     //We need to prevent the rounding that is occurring here!
 
@@ -385,7 +385,7 @@
 
 void CalculatorApplet::slotSubtractClicked()
 {
-    double currentValue = inputText.toDouble();
+    double currentValue = KGlobal::locale()->readNumber(inputText);
 
     if (previousAddSubOperation!=calcNone) {
         calculate(currentValue, previousAddSubOperation);
@@ -400,7 +400,7 @@
 
 void CalculatorApplet::slotMultiplyClicked()
 {
-    double currentValue = inputText.toDouble();
+    double currentValue = KGlobal::locale()->readNumber(inputText);
 
     if (previousMulDivOperation!=calcNone) {
         calculate(currentValue, previousMulDivOperation);
@@ -415,7 +415,7 @@
 
 void CalculatorApplet::slotDivideClicked()
 {
-    double currentValue = inputText.toDouble();
+    double currentValue = KGlobal::locale()->readNumber(inputText);
 
     if (previousMulDivOperation!=calcNone) {
         calculate(currentValue, previousMulDivOperation);
@@ -431,7 +431,7 @@
 void CalculatorApplet::slotEqualsClicked()
 {
     bool ok;
-    double currentValue = inputText.toDouble(&ok);
+    double currentValue = KGlobal::locale()->readNumber(inputText, &ok);
 
     if (ok == false) {
         handleError(i18n("ERROR"));
Index: applets/rssnow/plasma-applet-rssnow.desktop
===================================================================
--- applets/rssnow/plasma-applet-rssnow.desktop	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ applets/rssnow/plasma-applet-rssnow.desktop	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -1,6 +1,5 @@
 [Desktop Entry]
 Name=RSSNOW
-Name[it]=RSSORA
 Name[sr]=РСС‑одмах
 Name[sr@latin]=RSS‑odmah
 Name[x-test]=xxRSSNOWxx
Index: applets/luna/plasma-applet-luna.desktop
===================================================================
--- applets/luna/plasma-applet-luna.desktop	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ applets/luna/plasma-applet-luna.desktop	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -37,7 +37,7 @@
 Comment[ku]=Ji bo herêma xwe rewşa heyvê nîşan bide
 Comment[lv]=Rāda mēness fāzi
 Comment[nds]=De aktuelle Maanschiev wiesen
-Comment[nl]=Toont maanfasen voor u locatie
+Comment[nl]=Toont maanfasen voor uw locatie
 Comment[nn]=Vis korleis månen ser ut der du er
 Comment[pl]=Pokazywanie faz Księżyca w Twojej lokalizacji
 Comment[pt]=Mostrar as fases lunares para a sua localidade
Index: applets/showdesktop/plasma-applet-showdesktop.desktop
===================================================================
--- applets/showdesktop/plasma-applet-showdesktop.desktop	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ applets/showdesktop/plasma-applet-showdesktop.desktop	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -74,8 +74,8 @@
 Comment[ro]=Arată biroul Plasma
 Comment[ru]=Показать рабочий стол Plasma
 Comment[sl]=Prikaži namizje Plasma
-Comment[sr]=Баците поглед на Плазма површ
-Comment[sr@latin]=Bacite pogled na Plasma površ
+Comment[sr]=Баците поглед на плазма површ
+Comment[sr@latin]=Bacite pogled na plasma površ
 Comment[sv]=Visa Plasmas skrivbord
 Comment[th]=แสดงพื้นที่ทำงานของพลาสมา
 Comment[tr]=Plasma masaüstünü göster
Index: applets/konsoleprofiles/plasma-applet-konsoleprofiles.desktop
===================================================================
--- applets/konsoleprofiles/plasma-applet-konsoleprofiles.desktop	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ applets/konsoleprofiles/plasma-applet-konsoleprofiles.desktop	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -27,7 +27,7 @@
 Name[pl]=Profile Konsoli
 Name[pt]=Perfis do Konsole
 Name[pt_BR]=Perfis do Konsole
-Name[ro]=Profiluri consolă
+Name[ro]=Profiluri konsolă
 Name[ru]=Konsole: профили
 Name[sl]=Profili za Konzolo
 Name[sr]=Конзолини профили
@@ -55,7 +55,7 @@
 Comment[gl]=Enumera e inicia perfís de Konsole
 Comment[he]=הצג והפעל פרופילים של Konsole
 Comment[hu]=Indítható Konsole-profilok
-Comment[it]=Elenca ed avvia i profili di Konsole
+Comment[it]=Elenca ed esegue i profili di Konsole
 Comment[ja]=Konsole のプロファイルを起動します
 Comment[km]=រាយ និង​ចាប់ផ្ដើម​ទម្រង់​របស់ Konsole
 Comment[ku]=Profîlên Konsolê lîste bike û bide destpêkirin
Index: runners/konsolesessions/konsolesessions.desktop
===================================================================
--- runners/konsolesessions/konsolesessions.desktop	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ runners/konsolesessions/konsolesessions.desktop	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -27,7 +27,7 @@
 Name[pl]=Sesje konsoli
 Name[pt]=Sessões do Konsole
 Name[pt_BR]=Sessões do Konsole
-Name[ro]=Sesiuni consolă
+Name[ro]=Sesiuni konsolă
 Name[ru]=Konsole: сеансы
 Name[sl]=Seje Konzole
 Name[sr]=Конзолине сесије
Index: runners/spellchecker/plasma-runner-spellchecker_config.desktop
===================================================================
--- runners/spellchecker/plasma-runner-spellchecker_config.desktop	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ runners/spellchecker/plasma-runner-spellchecker_config.desktop	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -21,7 +21,7 @@
 Name[ga]=Feidhmitheoir an Litreora
 Name[gl]=Executor de comprobación da ortografía
 Name[he]=בודק האיות
-Name[it]=Lanciatore del controllo ortografico
+Name[it]=Esegue il controllo ortografico
 Name[ja]=スペルチェッカー
 Name[km]=កម្មវិធី​រត់​កម្មវិធីពិនិត្យអក្ខរាវិរុទ្ធ
 Name[ku]=Xebatkarê Kontrola Rastnivîsê
Index: runners/spellchecker/plasma-runner-spellchecker.desktop
===================================================================
--- runners/spellchecker/plasma-runner-spellchecker.desktop	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ runners/spellchecker/plasma-runner-spellchecker.desktop	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -14,7 +14,7 @@
 Name[ga]=Feidhmitheoir an Litreora
 Name[gl]=Executor de comprobación da ortografía
 Name[he]=בודק האיות
-Name[it]=Lanciatore del controllo ortografico
+Name[it]=Esegue il controllo ortografico
 Name[ja]=スペルチェッカー
 Name[km]=កម្មវិធី​រត់​កម្មវិធីពិនិត្យអក្ខរាវិរុទ្ធ
 Name[ku]=Xebatkarê Kontrola Rastnivîsê
@@ -53,7 +53,7 @@
 Comment[ga]=Seiceáil litriú focail
 Comment[gl]=Comproba a ortografía dunha palabra
 Comment[he]=בודק איות של מילה
-Comment[it]=Controlla l'ortografia delle parole
+Comment[it]=Controlla l'ortografia di una parola
 Comment[ja]=単語のスペルをチェックします
 Comment[km]=ពិនិត្យ​មើល​កា​រប្រកប​ពាក្យ
 Comment[ku]=Rastnivîsa peyvekê kontrol dike
Index: runners/contacts/plasma-runner-contacts.desktop
===================================================================
--- runners/contacts/plasma-runner-contacts.desktop	(.../tags/KDE/4.2.1/kdeplasma-addons)	(wersja 943487)
+++ runners/contacts/plasma-runner-contacts.desktop	(.../branches/KDE/4.2/kdeplasma-addons)	(wersja 943487)
@@ -53,7 +53,7 @@
 Comment[et]=Kirjete otsimine aadressiraamatust
 Comment[eu]=Zure helbide liburuan sarrerak aurkitzen ditu
 Comment[fi]=Etsii yhteystietoja osoitekirjasta
-Comment[fr]=Trouve des entrées dans votre carnet d'adresse
+Comment[fr]=Trouve des entrées dans votre carnet d'adresses
 Comment[ga]=Aimsigh teagmhálacha i do leabhar seoltaí
 Comment[gl]=Encontra entradas no seu caderno de enderezos
 Comment[he]=מצא רשומות מתוך האנשי קשר שלך
