diff -ur kdeplasma-addons-4.2.87svn969966/applets/pastebin/backends/pastebincom.cpp kdeplasma-addons-4.2.87svn969966.new/applets/pastebin/backends/pastebincom.cpp
--- kdeplasma-addons-4.2.87svn969966/applets/pastebin/backends/pastebincom.cpp	2009-02-26 10:13:10.000000000 +0100
+++ kdeplasma-addons-4.2.87svn969966.new/applets/pastebin/backends/pastebincom.cpp	2009-05-26 23:35:14.059626384 +0200
@@ -25,7 +25,7 @@
 PastebinCOMServer::PastebinCOMServer(KConfigGroup config)
     : PastebinServer()
 {
-    m_server = config.readEntry("pastebincom", "http://pastebin.com");
+    m_server = config.readEntry("pld.pastebincom", "http://pld.pastebin.com");
 }
 
 PastebinCOMServer::~PastebinCOMServer()
Tylko w kdeplasma-addons-4.2.87svn969966.new/applets/pastebin/backends: pastebincom.cpp~
